packages/shared/src/hooks/useLeaveSquad.ts

```diff
@@ -21,7 +23,11 @@ type UseLeaveSquadProps = {
 export const useLeaveSquad = ({ squad }: UseLeaveSquadProps): UseLeaveSquad => {
   const { logEvent } = useContext(LogContext);
   const { showPrompt } = usePrompt();
-  const { deleteSquad: deleteCachedSquad } = useBoot();
+  const { mutate } = useMutation(leaveSquad, {
+    onSuccess: () => {
+      persistedQueryClient.invalidateQueries({ queryKey: [RequestKey.Squads] });
```

Comment:
Can we also mutate the persistent client instead of just invalidating it?

Reply:
Hm, not quite sure I understand the question. 😅

You mean use the setQueryData function to update the cache instead of refetching?